.circleci/config.yml

```diff
@@ -119,6 +126,33 @@ jobs:
           failure_message: "**${CIRCLE_USERNAME}**'s WASM build failed. Please check why! 😭"
           webhook: "${DISCORD_STATUS_WEBHOOK}"
 
+  tests:
+    docker:
+      - image: cimg/rust:1.72.0
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run:
+          name: Generate lockfile
+          command: |
+            cd modules
+            cargo generate-lockfile
+      - restore_cache:
+          keys:
+            - cargocache-v2-tests-rust:1.72.0-{{ checksum "modules/Cargo.lock" }}
+      - run:
+          name: Modules tests
+          command: |
+            cd modules
+            cargo test --locked --all-targets -j 2 # Manually enable features if required, osmosis test tube requires golang to function
```

Comment:
it will take eternity with 2 compilation jobs or OOM if we pass more

Reply:
@CyberHoward 11 mins for tests and another 10 mins for saving cache. Is it fine?

Reply:
OMM?

Reply:
> OMM?

Out of memory

Reply:
> Also can't we use nextest?

We can, but why? We will have to install it first, which most likely will take more time than it will save

Reply:
Unless it's cached job and ran N(pretty sure 2 digit) times. Tests itself not slow but compilation of modules is

Reply:
> > Also can't we use nextest?
> 
> We can, but why? We will have to install it first, which most likely will take more time than it will save

Because nextest is way faster. Can use pre-built bin: https://nexte.st/book/pre-built-binaries#linux-x86_64